pyedgar/pyedgar.conf

```diff
@@ -59,6 +59,10 @@ KEEP_ALL=False
 ; This will be ignored if KEEP_ALL=True
 KEEP_REGEX=10-[KQ]|10[KQ]SB|8-K|(?:14A$)|13[FDG]
 
+; OVERWRITE_ON_EXTRACT controls whether to overwrite existing filings when extracting from feeds
+; Default is False to avoid re-extracting already downloaded filings
+OVERWRITE_ON_EXTRACT=False
```

Comment:
Move config file value to [Paths] section, under CACHE_FEED, and call it CACHE_FEED_OVERWRITE.